rules/import-style.js

```diff
@@ -183,6 +184,25 @@ const create = context => {
 			node,
 			messageId: MESSAGE_ID,
 			data,
+			fix(fixer) {
+				if (!autoFix) {
+					return;
+				}
+
+				if (node.type === 'ImportDeclaration' && allowedImportStyles.has('namespace')) {
+					const importedNames = node.specifiers
+						.filter(s => s.type === 'ImportSpecifier' || s.type === 'ImportDefaultSpecifier')
+						.map(s => s.local.name);
+
+					if (importedNames.length > 0) {
+						const namespaceIdentifier = moduleName === 'react' ? 'React' : importedNames[0];
+						return fixer.replaceText(
+							node,
+							`import * as ${namespaceIdentifier} from "${moduleName}"`,
```

Comment:
We need find all references and replace them with `namespace.x`

Reply:
I'll look into this. If you know the solution, please feel free to share it with me.

Reply:
You need

1) give a name for namespace import, it should be based on package name or filename
2) find all references, and check that variable name for the module are available in their scope.[this](https://github.com/sindresorhus/eslint-plugin-unicorn/blob/4e539b4b084ce06165a0fdbc02d309ce7cf75c82/rules/utils/avoid-capture.js#L132) utility maybe helpful.
3) Replace all references.

Reply:
Fixed 👍